.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+
+jobs:
+  # Build the Docker image
+  build:
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build & test docker image
+          command: ./ci/travis_system_integration.sh
```

Comment:
Should this script be renamed if it's used by CircleCI too?

Reply:
Done